fhir-model/src/main/java/com/ibm/fhir/model/visitor/Visitor.java

```diff
@@ -1347,4 +1348,14 @@
     void visit(java.lang.String elementName, Year value);
     void visit(java.lang.String elementName, YearMonth value);
     void visit(java.lang.String elementName, ZonedDateTime value);
+
+    /**
+     * @return the coordinates for the lat/long. 
+     * 
+     * @param elementName
+     * @param elementIndex
+     * @param location
```

Comment:
this should match the name of the parameter

Reply:
fixed in code generator.  I modified this class by hand (as you can see above).  I have fixed this case.